examples/apps/src/ble_bas_peripheral_auth.rs

```diff
@@ -0,0 +1,237 @@
+use embassy_futures::join::join;
+use embassy_futures::select::{select, Either};
+use embassy_time::Timer;
+use embedded_hal_async::digital::Wait;
+use rand_core::{CryptoRng, RngCore};
+use trouble_host::IoCapabilities;
+use trouble_host::prelude::*;
```

Comment:
Please include the IoCapabilities in the prelude.

Reply:
Done